packages/task/src/node/task-server.ts

```diff
@@ -171,15 +171,15 @@ export class TaskServerImpl implements TaskServer, Disposable {
     protected fireTaskExitedEvent(event: TaskExitedEvent, task?: Task): void {
         this.logger.debug(log => log('task has exited:', event));
 
-        this.clients.forEach(client => {
-            client.onTaskExit(event);
-        });
-
-        if (task && task instanceof ProcessTask && task.processType === 'process') {
+        if (task && task instanceof ProcessTask) {
```

Comment:
`if (task instanceof ProcessTask)` should be enough.

Reply:
I will change it.
However, it is the same as in `fireTaskCreatedEvent` method. Should I change it there too?